src/components/field/ShareScreen.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import i18n from '@dhis2/d2-i18n'
+import PropTypes from '@dhis2/prop-types'
+import { CheckboxField } from './CheckboxField'
+
+const CODE = 'allowScreenCapture'
+
+export const defaultShareScreen = false
+
+export const ShareScreen = ({ value, onChange, ...props }) => {
+    const handleCheckbox = e => {
+        onChange({ ...value, [CODE]: e.checked })
+    }
+
+    return (
+        <CheckboxField
+            name={CODE}
+            label={i18n.t('Take screenshots and share it')}
+            helpText={i18n.t(
+                'Allow user to take screenshots and share their screen.'
+            )}
+            checked={value[CODE]}
+            onChange={handleCheckbox}
+            {...props}
+        />
```

Comment:
Since the help text is pretty descriptive and short, I think we can use it as the label:

```suggestion
        <CheckboxField
            name={CODE}
            label={i18n.t(
                'Allow user to take screenshots and share their screen.'
            )}
            checked={value[CODE]}
            onChange={handleCheckbox}
            {...props}
        />
```